compose/install.md

```diff
@@ -228,42 +163,12 @@ Pre-releases built from the "master" branch are also available for download at
 
 ----
 
-## Upgrading
-
-If you're upgrading from Compose 1.2 or earlier, remove or
-migrate your existing containers after upgrading Compose. This is because, as of
-version 1.3, Compose uses Docker labels to keep track of containers, and your
-containers need to be recreated to add the labels.
-
-If Compose detects containers that were created without labels, it refuses
-to run, so that you don't end up with two sets of them. If you want to keep using
-your existing containers (for example, because they have data volumes you want
-to preserve), you can use Compose 1.5.x to migrate them with the following
-command:
-
-```console
-$ docker-compose migrate-to-labels
-```
-
-Alternatively, if you're not worried about keeping them, you can remove them.
-Compose just creates new ones.
-
-```console
-$ docker container rm -f -v myapp_web_1 myapp_db_1 ...
-```
-
 ## Uninstallation
 
 To uninstall Docker Compose if you installed using `curl`:
```

Comment:
Should this section only apply to Linux systems, maybe?

Reply:
Right, I didn't changed the existing page layout.